src/main/kotlin/io/sip3/salto/ce/rtpr/RtprHandler.kt

```diff
@@ -166,27 +176,28 @@ open class RtprHandler : AbstractVerticle() {
         }
     }
 
-    private fun sessionId(address: Address): Long {
-        return (IpUtil.convertToInt(address.addr).toLong() shl 32) or (address.port and 0xfffe).toLong()
+    private fun sdpSessionId(address: Address): Long {
+        return sdpSessionId(address.addr, address.port)
     }
 
     private fun terminateExpiredSessions() {
         val now = System.currentTimeMillis()
-
-        sdp.filterValues { it.timestamp + aggregationTimeout < now }
-                .forEach { (key, _) -> sdp.remove(key) }
+        val expiredCallIds = mutableSetOf<String>()
```

Comment:
Is't it just cleaner to do `sdp.remove(report.callId)` every time. Way less code